### PR TITLE
Script output logging for e2e tests

### PR DESCRIPTION
**Pull request by radoslawc**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug
> /kind cleanup

/kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
This will allow tracing progress of e2e test while logged in the testing VM without relaying on Prow's logs
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: e2e/lib/testing.sh [hunk 1]
```diff
@@ -12,6 +12,8 @@
 # shellcheck source=e2e/lib/_utils.sh
 source "${E2EDIR:-$HOME/test-infra/e2e}/lib/_utils.sh"
 
+LOG_DIR="/var/log/e2e"
```
Comment (efiacor) [inline e2e/lib/testing.sh:15]: Looks like a permissions issue on this path

Comment (radoslawc) [reply e2e/lib/testing.sh]: @efiacor you're right, good catch, thanks

**Comment by efiacor** [comment]
home/ubuntu/test-infra/e2e/lib/testing.sh: line 50: ARTIFACTS: unbound variable

**Comment by radoslawc** [comment]
/override e2e-free5gc-ubuntu-focal 
failing on unrelated to this PR longstanding issue

**Comment by radoslawc** [comment]
> home/ubuntu/test-infra/e2e/lib/testing.sh: line 50: ARTIFACTS: unbound variable

fixed, thanks

**Pull request merged**